cves/kernel/CVE-2020-29369.yml

```diff
@@ -106,19 +110,16 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
-  note: Discovered automatically by archeogit.
-- commit: 615d6e8756c87149f2d4c1b93d471bca002bd849
-  note: Discovered automatically by archeogit.
-- commit: 146425a316fb937fbdcac018b34a23c67d12214b
-  note: Discovered automatically by archeogit.
+  note: |
+    Discovered automatically by archeogit. Manually confirmed.
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
 
   For the second round of reviewing, you will be giving a certain amount of
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes:
+upvotes: 2
```

Comment:
1 upvote (sorry this is my last one but id give more if i could)

Reply:
added!